pkg/acsengine/const.go

```diff
@@ -90,6 +90,7 @@ var KubeImages = map[string]map[string]string{
 		"addonmanager":    "kube-addon-manager-amd64:v6.4-beta.2",
 		"dnsmasq":         "k8s-dns-dnsmasq-amd64:1.14.4",
 		"pause":           "pause-amd64:3.0",
+		"tiller":          "tiller:v2.5.1",
```

Comment:
> Let's replace the "tiller:v2.5.1" string literal with a const, like DefaultTillerImage (or something) here and in all the k8s versions below. 

Oh, I like that. Would opt to get that refactoring in in a subsequent PR, as we're following the existing pattern. New PR would have the refactoring, and maybe some new unit tests (whoa!).